cmd/activator/main.go

```diff
@@ -135,14 +136,26 @@ func (a *activationHandler) handler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, msg, int(status))
 		return
 	}
+
+	var transport http.RoundTripper
+	if endpoint.IsVerified() {
+		transport = http.DefaultTransport
+		if r.ProtoMajor == 2 {
+			transport = h2cutil.NewTransport()
+		}
+	} else {
+		transport = retryRoundTripper{
```

Comment:
The PR is meant to address not retrying HTTP POSTs it's still possible here if the endpoint isn't verified.

Reply:
The [PR comment](https://github.com/knative/serving/pull/1665#issue-203631013) refers to this fact. This issue is being resolved with a [multi PR approach](https://github.com/knative/serving/issues/1448#issuecomment-406070701)

This initial PR avoids HTTP retries by activator **when the user has defined an HTTPGet readinessProbe**. The follow up PR will default to queue-proxy health check when user has not defined a readinessProbe

Reply:
To clarify my comment if the endpoint `VerifiedStatus` is `Failed` it will go into the else block and use the retryTripper for HTTP POSTs